src/MPI_with_SYCL/send_recv_buff.cpp

```diff
@@ -66,19 +100,16 @@ int main(int argc, char *argv[]) {
         auto kern = [=](sycl::id<1> id) { acc[id] *= 2; };
         h.parallel_for(sycl::range<1>{nelem}, kern);
       };
-      // When using buffers with CUDA-aware MPI, a host_task must be used with a
-      // sycl::interop_handle in the following way. This host task command group
-      // uses MPI_Send to send the data to rank 1.
+      // When using buffers with device-aware MPI, a host_task must be used with
+      // a sycl::interop_handle in the following way. This host task command
+      // group uses MPI_Send to send the data to rank 1.
       auto ht = [&](sycl::handler &h) {
         sycl::accessor acc{buff, h};
         h.host_task([=](sycl::interop_handle ih) {
-          // get the native CUDA device pointer from the SYCL accessor.
-          auto cuda_ptr = reinterpret_cast<int *>(
-              ih.get_native_mem<sycl::backend::ext_oneapi_cuda>(acc));
-
+          void *device_ptr = getDevicePointer(acc, ih);
           MPI_Status status;
```

Comment:
```
/__w/SYCL-samples/SYCL-samples/src/MPI_with_SYCL/send_recv_buff.cpp:110:22: error: unused variable 'status' [-Werror,-Wunused-variable]
  110 |           MPI_Status status;
      |                      ^~~~~~
```
The issue was there before, but we weren't building this example in CI (because it required CUDA). Would you mind removing the unused variable?

Reply:
Cheers, done